openapi-cli/src/test/resources/ballerina-to-openapi/constraint/decimalMax.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/http;
```

Comment:
Lets add the license header [1] to sample codes. Check in other samples as well.

[1] - https://github.com/ballerina-platform/module-ballerina-websub/blob/1b17fd1a1a78774e7eec2bee7da25b41329b7291/ballerina/annotation.bal#L1